pkg/operator/starter.go

```diff
@@ -82,6 +83,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 			"csidriver.yaml",
 			"node_sa.yaml",
 			"cabundle_cm.yaml",
+			"volumesnapshotclass.yaml",
```

Comment:
@dobsonj I'm currently working on a ticket to only deploy this CR if the CRD exists: https://github.com/openshift/csi-driver-manila-operator/pull/159/files

Could you take the chance and introduce those changes in this PR as well? Sorry...

Reply:
Yeah of course, no problem :)